src/qt/font.cpp

```diff
@@ -346,13 +399,19 @@ wxFontFamily wxFont::DoGetFamily() const
 
 void wxNativeFontInfo::Init()
 {
+    m_wxFontWeight = wxFONTWEIGHT_NORMAL;
 }
 
 float wxNativeFontInfo::GetFractionalPointSize() const
 {
     return m_qtFont.pointSizeF();
 }
 
+wxSize wxNativeFontInfo::GetPixelSize() const
+{
+    return wxSize( m_qtFont.pixelSize(), m_qtFont.pixelSize() );
```

Comment:
I don't think this is correct/consistent with the other ports, if the pixel size is the height of the font (as I think it is), it should be used for `y` component only and 0 should be returned as `x`, meaning "unknown".

BTW, have you seen the comment in `wxFontBase::operator==()` in `src/common/fontcmn.cpp`? It probably is out of date and the check for `__WXQT__` can be removed from there.

Reply:
I agree with your suggestion here, I was uncertain the best way to tackle this and this approach was the best I could think of at the time. I'll update this to match your suggestion!

I haven't checked out the comment you specified in `src/common/fontcmn.cpp`, but I will check it out and remove the check you specified if appropriate.